### ISSUE TITLE
Why is the owned state always false when listening to the loaded event?

### ISSUE THREAD

**Issue by mcfarljw** [j3k0/cordova-plugin-purchase#547]
I've noticed that after a non-consumable purchase is approved and `product.finish()` is called the next time the app loads owned is set to false. The following examples demonstrates the code after a purchase and app restart:

```
store.register({
  id: 'remove_ads',
  type: store.NON_CONSUMABLE
});

store.when('remove_ads').loaded(
  function(product) {
    console.log('remove_ads:', product.owned); // shows false
  }
);

store.refresh();
```

The only way to get the owned state after a purchase is to call `product.finish()` and listen for the owned state every single time the app loads. For example:

```
store.register({
  id: 'remove_ads',
  type: store.NON_CONSUMABLE
});

store.when('remove_ads').updated(
  function(product) {
    if (product.state === 'approved') {
      product.finish();
    }
    if (product.state === 'owned') {
      console.log('remove_ads:', product.owned); // shows true
    }
  }
);

store.refresh();
```

This seems a bit awkward. If an item is owned and loaded from the server should't the owned state just be true? Why does the approved event fire every single time and need to call product finish just to get the true owned state?

**Comment by mcfarljw** [comment]
This appears to be related to https://github.com/j3k0/cordova-plugin-purchase/issues/532 but I wanted to add some more complete complete reproduction steps.

**Comment by Dexus** [comment]
closed for duplicate of #532
